WDAC-Policy-Wizard/app/Resources/policyTemplates/Recommended_Driver_Blocklist.xml

```diff
@@ -1,16 +1,30 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <SiPolicy xmlns="urn:schemas-microsoft-com:sipolicy">
-  <VersionEx>10.0.25210.0</VersionEx>
+  <VersionEx>10.0.27685.0</VersionEx>
   <PlatformID>{2E07F7E4-194C-4D20-B7C9-6F44A6C5A234}</PlatformID>
   <Rules>
     <Rule>
       <Option>Enabled:Unsigned System Integrity Policy</Option>
     </Rule>
+    <Rule>
```

Comment:
These rules should not be added to the template. It will cause issues when the Wizard merges these rules with a customer's policy

Reply:
Sounds great Jordan! I'll remove all rules except Unsigned SIP to match the way the policy was before :)